AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -564,21 +564,21 @@
  * Text File View Controller
  */
 "createtextfile.title" = "Create Text File";
-"createtextfile.dismiss.confirmation.title" = "Dismiss?";
-"createtextfile.dismiss.confirmation.message" = "Are you sure you want to cancel the creation of this text file?\n\nAll work will be discarded.";
-"document.edit.button.discard" = "Discard Changes";
+"createtextfile.dismiss.confirmation.title" = "Cancel";
+"createtextfile.dismiss.confirmation.message" = "If you cancel the creation of this text file the work won't be saved.";
+"document.edit.button.discard" = "Cancel";
```

Comment:
Fixed.

I've gone with "Don't Save" as I think it's clearer and more accurate.